### PR TITLE
ADDS PURPLE SANS FROM UNDERTALE

### PR DESCRIPTION
**Pull request by BurgerLUA**
PR for porting Phoron Restructurant Humans from Persistence Station.

Everything is done except for making it so that Phoron Lads don't get hungry, but I can easily do that later. This changes a few things in the code too to make Phoron Lads work as it there was a lot of shitcode.

Just needs to lore for now. If loredevs don't step forward and add lore I'm just going to use the default lore from the species description page from Persistence Station.

**Edit:**
Corrections made by Carl, the big bad.
**Edit edit:**
Changed it to the actual in-game name, by Jade.

### CODE REVIEW

**Comment by Sparrowbird611** [comment]
Nice.

**Comment by SilvernKell** [comment]
i love you

**Comment by BurgerLUA** [comment]
Hey just so everyone is aware and so people can stop spreading misinformation: The plasmapeople in this PR aren't skeletons. They're humans who've been exposed to phoron in a rare way to cause their bones and flesh to solidify into a phoron-calcium composite. They don't actually look too much like skeletons so it's not entirely magic.

**Comment by bearrrrrrrr** [comment]
we uhhhhh don't have a lore dev yet so please don't merge this until we get a lore dev to approve this

love u burger <3

**Comment by AdamElTablawy** [comment]
Hey burger, if a plasmaman catches fire will it heat up the room?

Because that's gonna cause issues

**Comment by BurgerLUA** [comment]
> Hey burger, if a plasmaman catches fire will it heat up the room?
> 
> Because that's gonna cause issues

fuck if I know, probably

just don't catch on fire smh

**Comment by AdamElTablawy** [comment]
@BurgerLUA that'd be a meme because the eternally burnin plasmaman would eventually heat the room to a ridiculous level

instead just give it a fire overlay and raise its body temperature significantly or something

**Comment by BurgerLUA** [comment]
@AdamElTablawy I like none of these suggestions. I'll do something to stop it if it's an issue.

**Comment by BurgerLUA** [comment]
There is a bug that causes them to die that carl told me about so don't testmerge.

**Comment by ProbablyCarl** [comment]
Suits are far, far too vulnerable, considering you take such excessive amounts of burn damage, you die in under a minute and can't put it out without taping/welding up the suit.
A single hit opens a tiny puncture. A few consecutive hits and you turn into a fireball.
A suggestion to fix this would be adding the auto-extinguishers present on Para suits.

As it stands, fighting maint mobs is a guarantee of death if you're not quick with a roll of tape.

Or we could just give them something akin to a lightly armored hardsuit, which'd make far more sense. Given they need the atmosphere held within to not drop dead.
It'd also make them something far more unique.

**Comment by ProbablyCarl** [comment]
Additionally, they're unable to wear any sort of goggles, alongside gloves. Perhaps giving them a manner of hud for their helmet, dependent on job, would suffice?

As for the gloves, I don't know if the suit is sufficient.

**Comment by wojciechjedrzejak** [comment]
No gloves means no insuls so it isn't

**Comment by Don2137** [comment]
No gloves means no insuls so not really

**Comment by Don2137** [comment]
Also, suits don't have correct resistances for their jobs, especially engineering ones. The kit thingy should probably check for ID to prevent powergaming.

**Comment by Don2137** [comment]
Also, the Head of Security suit is missing from the adapter kit

File: code/modules/species/station/plasmasans.dm [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+	preview_icon = 'icons/mob/human_races/species/plasmasans/preview.dmi'
+	bandages_icon = 'icons/mob/bandage.dmi'
+
+	hidden_from_codex = FALSE
+
+	unarmed_types = list(/datum/unarmed_attack/stomp, /datum/unarmed_attack/punch)
+
+	breath_type = GAS_PHORON
+	poison_types = list(GAS_OXYGEN = TRUE) //Getting oxygen into your lungs HURTS
+	exhale_type = GAS_HYDROGEN
```
Comment (chaoko99) [inline code/modules/species/station/plasmasans.dm:17]: So fun fact: They will continue to exhale hydrogen even while on internals.

I see no problems with this. It'll be hilarious.

Comment (ProbablyCarl) [reply code/modules/species/station/plasmasans.dm:17]: Yee. It's pretty great.

**Comment by ProbablyCarl** [comment]
Works fine. Just waiting on the above to be corrected.

File: code/modules/species/station/plasmasans.dm (hunk 1 above)
Comment (chaoko99) [reply code/modules/species/station/plasmasans.dm:17]: They're also gonna exhale plasma as well.

**Comment by Dark1Star3s** [comment]
The fact that they are called Plasmasans. Is kind of stupid... can we actually take it somewhat seriously can call them something right in code? Also, I would LOVE to have Paradise's fire-extinguisher for all their suits to be ported over, including the refills, and rather have the suits they wear be uniforms, rather than suits like on Paradise. Which allows them to wear armor.

**Comment by BurgerLUA** [comment]
It doesn't matter what they're called in code.

**Comment by chaoko99** [comment]
It does. When someone inevitably goes to search it in the future, it'll cause them issues.

**Comment by BurgerLUA** [comment]
It literally won't, they can just look at the species files. It makes 0 difference as they aren't call plasmamen in game either.

**Comment by ProbablyCarl** [comment]
We've had these test merged for a bit, and aside from Travis throwing a shit fit, and a few missing QoL changes, they're working just fine.
Just let me know when it's good to merge - or, hell, just merge it yourself.

**Comment by BurgerLUA** [comment]
Travis just needs to be fixed and it will be good. It will be ready by Monday.

**Pull request merged**